docs/DevOps.md

```diff
@@ -73,7 +73,7 @@ Any warnings or errors will be printed out to the console.
 
 Here are the steps to create a new release.
 
-1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/address/MainApp.java).
+1. Update the version number in [`MainApp.java`](https://github.com/se-edu/addressbook-level3/tree/master/src/main/java/seedu/telegramHandle/MainApp.java).
```

Comment:
This seems inaccurate - the current file structure does not reflect such a filepath.

(This is also repeated in other files such as DeveloperGuide.md, so be wary if a fix is to be done here.)